section2/reflection.md

```diff
@@ -1,15 +1,22 @@
 ## Section 2 Reflection
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
-
+I like the idea that mistakes are a part of learning and not a distraction from it. Also, asking for help makes you a better
+programmer.
 2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
-
+I demonstrate it by setting measurable goals. Positive progression over perfection. I am also open to feedback and suggestions.
+I'm not great at asking for help and will struggle with a problem alone longer than necessary.
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
-
+I liked practicing running node without directions. I did my best to recall how to do it from memory. I still had to look it up,
+but I was close! It's fun to practice the things I learn as I learn them. I'm now glad to read that the directions were intentionally vague. I got confused and it definitely made me uncomfortable. I went to study hall, and another student was stuck
+on the same question. That made me feel better knowing I wasn't the only one who didn't fully understand.
```

Comment:
I love so much that y'all utilized the study halls and were together being stuck (I don't want you to be painfully stuck, but that you got to see you weren't the only one!)